Demo/AuthenticatorDemoUITests/GoogleSignInSafariScreen.swift

```diff
@@ -0,0 +1,73 @@
+import ScreenObject
+import XCTest
+
+struct GoogleSignInSafariScreen {
+
+    private let app: XCUIApplication
+    private let email = APICredentials.Tests.GoogleAccount.email
+    private let password = APICredentials.Tests.GoogleAccount.password
+
+    enum Labels {
+        static let selectDifferentAccount = "Use another account"
+        static let next = "Next"
+    }
+
+    init(app: XCUIApplication) {
+        self.app = app
+    }
+
+    func authenticate(file: StaticString = #file, line: UInt = #line) {
+        // The device in which we're running the tests might have pre-existing accounts available.
+        let emailField = app.staticTexts[email]
+        let selectDifferentAccount = app.staticTexts[Labels.selectDifferentAccount]
+        if emailField.waitForExistence(timeout: 5) {
+            emailField.tap()
+        } else {
+            if selectDifferentAccount.waitForExistence(timeout: 5) {
+                selectDifferentAccount.tap()
+            }
+            typeEmail()
+            typePassword()
+        }
+
+        // To make sure the credentials input screen is gone, check none of the input elemets are
+        // on screen.
+        XCTAssertFalse(app.textFields.firstMatch.exists, file: file, line: line)
+        XCTAssertFalse(app.secureTextFields.firstMatch.exists, file: file, line: line)
+    }
+
+    private func typeEmail() {
+        app.textFields.firstMatch.typeText(email)
```

Comment:
For some reason, unless I tapped into the input before entering text, the text was not entered (might this be the reason for fails on CI 🤔 ?):

```suggestion
        let emailInput = app.textFields.firstMatch
        emailInput.tap()
        emailInput.typeText(email)
```

Reply:
🤦‍♂️ 

So what happened here is that I run the tests on the Simulator I use for regular development, where I don't have the software keyboard enabled so I can type in it directly from my Mac's keyboard without clicking the individual keys in the on screen one. However, that's not how Simulators run by default.

I realized it when testing on a different Simulator (TIL, always run the UI tests on a few different sims). Adding the tap, as you suggested, solved it for me, too.

See c7fae528